modules/340-monitoring-kubernetes/openapi/config-values.yaml

```diff
@@ -3,10 +3,18 @@ properties:
   ebpfExporterEnabled:
     type: boolean
     default: true
+    x-doc-deprecated: true
     x-examples:
     - false
     description: |
-      Set to 'false' to disable ebpf_exporter in case it behaves incorrectly.
+      Does not affect the operation of module components.
+  oomKillExporterEnabled:
```

Comment:
`oomKillExporterEnabled` is defined in the config values, but  helm templates use a different key - `oomKillsExporter` . now daemonset will never be rendered., which key is correct?
https://github.com/deckhouse/deckhouse/pull/10188/files#diff-ed0c292d1a1e9ef4dde701ca5afaee5e7a6bac2994d080074ba47d3a44d24542R7

Reply:
fixed